index.html

```diff
@@ -4,35 +4,80 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css"/>
```

Comment:
In HTML5 self-closing tags omit the forward-slash at the end "/" 
`<link rel="stylesheet" href="./style.css" >`